packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass

```diff
@@ -106,9 +106,11 @@
     text-align: center
     margin: $calendar-weekly-day-label-margin
 
-    .v-btn
+    .v-icon-btn
       font-size: $calendar-weekly-day-label-font-size
-      text-transform: none
+
+      &:focus > .v-icon-btn__overlay
+        opacity: calc(.25 * var(--v-theme-overlay-multiplier))
```

Comment:
I don't think this was an intentional design in v2, :focus-visible just didn't exist yet.

Reply:
Good point
leaving only font size